bindings/python/ParametersHandler/src/YarpParametersHandler.cpp

```diff
@@ -28,7 +30,14 @@ void CreateYarpParameterHandler(pybind11::module& module)
                std::shared_ptr<YarpImplementation>,
                IParametersHandler>(module, "YarpParametersHandler")
         .def(py::init())
-        .def("set_from_file", &YarpImplementation::setFromFile, py::arg("Filename"));
+        .def("set_from_filename",
+             [](YarpImplementation& impl, const std::string& filename) -> bool {
+                 const auto filePath
+                     = yarp::os::ResourceFinder::getResourceFinderSingleton().findFileByName(
+                         filename);
+                 return impl.setFromFile(filePath);
+             })
```

Comment:
Not a big problem, but inserting actual logic in the pybind11 code instead of leaving the logic on the C++ Side and just keep pybind11 code to contain glue code may be a dangerous path, especially if in the future we amt/hope to generate automatically the pybind11 glue code.

Reply:
I open an issue for this :)